src/sentry/src/Tracing/Aspect/ViewRenderAspect.php

```
@@ -0,0 +1,51 @@
+    public function process(ProceedingJoinPoint $proceedingJoinPoint)
+    {
+        if (! $this->feature->isTracingSpanEnabled('view')) {
+            return $proceedingJoinPoint->process();
+        }
+
+        $arguments = $proceedingJoinPoint->arguments['keys'] ?? [];
+        $template = $arguments['template'] ?? 'unknown';
+        $data = $arguments['data'] ?? [];
+
+        return trace(
+            fn (Scope $scope) => $proceedingJoinPoint->process(),
```

PHPMD (2.15.0)

Avoid unused parameters such as '$scope'. (undefined)

(UnusedFormalParameter)